guide/english/css/properties/background-position-property/index.md

```diff
@@ -8,7 +8,7 @@ The background-property sets the position from where the background image should
 ```css
 /* setting background-image of HTML doc */
 body {
-  background-image: url('https://i.imgur.com/6Z2VStD.png');
+  background-image: url('https://cdn-media-1.freecodecamp.org/imgr/6Z2VStD.png');
```

Comment:
@raisedadead Do we even need this? I think we can do away with this link altogether + the image does not make sense as a background

Reply:
@thecodingaviator Thank you for the catch.
Then we need another photo to replace. do you have anything in mind?
We could also differ content related modifications to another PR.
So it is you call.

Reply:
Pick up any photograph from unsplash. I think this one looks good: https://unsplash.com/photos/ZXPDzz9ax1w